### PR TITLE
Add integration tests for web cmd

### PR DESCRIPTION
**Pull request by carlosms**
Part of #232.
Tests for `web sql`, `web parse`.

### CODE REVIEW

File: cmd/srcd/cmd/web_test.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+}
+
+func (s *WebTestSuite) testCommon(subcmd string, assertions func(url string)) {
+	require := s.Require()
+
+	var out bytes.Buffer
+
+	command := s.CommandContext(context.TODO(), "web", subcmd)
+	command.Stdout = &out
+	command.Stderr = &out
```
Comment (dpordomingo) [inline cmd/srcd/cmd/web_test.go]: why not writing `stderr` into a new error buffer, and then assert that if there were no errors, there is nothing written in the error buffer?

Comment (carlosms) [reply cmd/srcd/cmd/web_test.go]: We are sending the log messages to `stderr`. We could simply ignore `stderr` and capture `stdout` only, but I left it combined to work in a consistent way with `RunCommand`.

Comment (dpordomingo) [reply cmd/srcd/cmd/web_test.go]: errors in `stdout` :(
ok...
given that constraint, consistency was a good decision :+1:

Comment (carlosms) [reply cmd/srcd/cmd/web_test.go]: No, error messages also go to `stderr`.

**Pull request merged**